cmd/binfmt/main.go

```diff
@@ -148,7 +165,7 @@ func run() error {
 		defer syscall.Unmount(mount, 0)
 	}
 
-	for _, name := range parseArch(toUninstall) {
+	for _, name := range parseUninstall(toUninstall) {
```

Comment:
this `arm64` special case should probably handled in parse now. I feel like the `-foo` suffix check can also be simplified but not a blocker.